tests/__tests__/jestconfig-package.spec.ts

```diff
@@ -1,13 +1,13 @@
+const pkgDir = require('pkg-dir');
+import { getJestConfig } from '../../src/utils';
+
 describe('get package json config', () => {
-  const pkgDir = require('pkg-dir');
   let yargsMock;
-  let getJestConfig;
 
   beforeEach(() => {
     jest.resetModules();
     yargsMock = jest.fn();
     jest.setMock('yargs', yargsMock);
-    getJestConfig = require('../../src/utils').getJestConfig;
```

Comment:
It does not. I don't even think multiple requires do anything state-wise. Simply an oversight.